### PR TITLE
feat: listing out videos on video-listing-page

### LINKED ISSUE #8: add: video listing page

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I listed down all videos by fetching API. #9 
Screenshot (1349)

### CODE REVIEW

File: src/Context/MainContext/MainContext.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const {
+          data: { videos },
+        } = await axios.get("/api/videos");
+        setVideos(videos);
+      } catch {
+        console.error("Error while fetching data");
```
Comment (mariyasada) [inline src/Context/MainContext/MainContext.jsx:31]: liked you destructured here

File: src/backend/db/videos.js [hunk 2]
```diff
@@ -1,22 +1,178 @@
...
+    img: "./assets/thriller1.jpg",
+    title: "Eleven & Mike's Cutest Moments | Stranger Things S1-3",
+    owner: "Still Watching Netflix",
+    views: "8.9M",
+    date: "Jul 10, 2019",
+    categoryName: "Thriler",
+  },
+  {
+    _id: uuid(),
+    img: "./assets/thriller2.jpg",
```
Comment (mariyasada) [inline src/backend/db/videos.js]: fix the typo thriler to thriller

**Review by mariyasada — APPROVED** [review]
the overall code structure is good. fix the one typo which I have mention

File: src/Components/VideoListingCard/VideoListingCard.css [hunk 3]
```diff
@@ -1,10 +1,17 @@
+.card-display-grid {
+  padding: 3rem;
+  display: grid;
+  gap: 2rem;
+  grid-template-columns: repeat(auto-fit, minmax(21rem, 1fr));
```
Comment (tushartiwari7) [inline src/Components/VideoListingCard/VideoListingCard.css]: i think `auto-fill`will work better here as when there are less products in a row you don't want them to be spreaded across the availaible space.

**Review by tushartiwari7 — APPROVED** [review]
Ui looks great,
And is responsive too.
added a suggestion,have a look though.

File: src/Components/VideoListingCard/VideoListingCard.css (hunk 3 above)
Comment (Bhakti199) [reply src/Components/VideoListingCard/VideoListingCard.css]: okay, I will try it out.

**Pull request merged**